### ISSUE TITLE
Error when running get_pseudobulk(): AttributeError: 'csr_matrix' object has no attribute 'A'

### ISSUE THREAD

**Issue by raquelgarza** [saezlab/decoupler-py#141]
Hi! 

I'm running your tool on a jupyter notebook (python 3.12, macos) using an anndata generated with scanpy (1.10.1)

I got this error message when running get_pseudobulk():
![image](https://github.com/saezlab/decoupleR/assets/31410823/35e4762b-3f30-4c92-b8f2-363bbc7cbfb1)

I solved it by commenting line 262 and 263 from decoupler/utils_anndata.py
```
                #if isinstance(X, csr_matrix):
                #    profile = profile.A
```
I thought I would report it in case this is due to a dependency update.

**Comment by PauBadiaM** [comment]
Hi @raquelgarza,

This is an error introduced by the new version of `scipy`. I will make a fix but in the meantime you can downgrade `scipy < 1.14.0`.

**Comment by PauBadiaM** [comment]
Fixed in 2192575a1672bfd665b4de209fb9e6af3e17528c, you can install this devel version by running:
```
pip install git+https://github.com/saezlab/decoupler-py

```

**Comment by PauBadiaM** [comment]
This issue should be solved with the new version of decoupler, it can be installed with:
```
pip install decoupler==1.8.0
```
